src/test/java/co/rsk/federate/adapter/ThinConverterTest.java

```diff
@@ -25,61 +68,87 @@ private co.rsk.bitcoinj.core.Sha256Hash createThinHash() {
         return co.rsk.bitcoinj.core.Sha256Hash.wrap(bytes);
     }
 
-    @Test
-    void toThinInstanceStoredBlock() {
-        org.bitcoinj.core.NetworkParameters params = org.bitcoinj.params.RegTestParams.get();
-        BigInteger chainWork = BigInteger.TEN;
-        int height = 200;
-        org.bitcoinj.core.Block originalBlock = new org.bitcoinj.core.Block(params, 1, createOriginalHash(), createOriginalHash(), 1000, 2000, 3000, new ArrayList<>());
+    @ParameterizedTest
+    @MethodSource("validChainWorkArgsProvider")
+    void toThinInstanceStoredBlock(BigInteger chainWork) {
+        // arrange
+        org.bitcoinj.core.Block originalBlock = new org.bitcoinj.core.Block(bitcoinCoreParams, 1, createOriginalHash(), createOriginalHash(), 1000, 2000, 3000, new ArrayList<>());
         org.bitcoinj.core.StoredBlock originalStoredBlock = new org.bitcoinj.core.StoredBlock(originalBlock, chainWork, height);
-        co.rsk.bitcoinj.core.StoredBlock thinStoredBlock = ThinConverter.toThinInstance(originalStoredBlock, new BridgeRegTestConstants());
+
+        // act
+        co.rsk.bitcoinj.core.StoredBlock thinStoredBlock = ThinConverter.toThinInstance(originalStoredBlock, bridgeConstants);
+
+        // assert
         assertEquals(chainWork, thinStoredBlock.getChainWork());
         assertEquals(height, thinStoredBlock.getHeight());
         assertArrayEquals(originalBlock.bitcoinSerialize(), thinStoredBlock.getHeader().bitcoinSerialize());
+        assertNull(ThinConverter.toThinInstance(null, bridgeConstants));
```

Comment:
why is this here? shouldn't this be in another test? You are not even using the param from the method source.

Reply:
You right. It can be on its own test.